Lib/test/test_pyrepl/test_pyrepl.py

```diff
@@ -1305,3 +1312,7 @@ def test_readline_history_file(self):
         output, exit_code = self.run_repl("exit\n", env=env)
         self.assertEqual(exit_code, 0)
         self.assertNotIn("\\040", pathlib.Path(hfile.name).read_text())
+
+    def test_keyboard_interrupt_after_isearch(self):
+        output, exit_code = self.run_repl(["\x12", "\x03", "exit"])
```

Comment:
Would it be possible to add (module-level) constants for "\x12" and "\x03"?

Reply:
We'll do that later as part of more tests for control characters.